### PR TITLE
My cart

### PR DESCRIPTION
**Pull request by theVivek-kumar**
mycart component done please review

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
code looks good ,  use var for colors it would be good

File: my-cart/mycart.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+:root {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: rgb(32, 32, 32);
+}
+
+.navbar-md-head {
+    display: flex;
+    justify-content: space-around;
+    background-color: #482880;
```
Comment (nisargpawade) [inline my-cart/mycart.css:20]: use variables for colors

File: my-cart/mycart.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                    <h4>$568</h4>
+                    
+                </div>
+                <div class="discount-chart">
+                    <h4>Discount</h4>
+                    <h4>-$89</h4>
+                
+                </div>
+                <div class="discount-chart">
+                    <h4>Dilivery Charge</h4>
```
Comment (nisargpawade) [inline my-cart/mycart.html:92]: delivery spelling

**Review by nisargpawade — APPROVED** [review]

File: my-cart/mycart.css (hunk 1 above)
Comment (manojsarna) [inline my-cart/mycart.css:20]: use variables for eg. --COLOR-BLACK

File: my-cart/mycart.css [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+
+}
+
+.fa-search {
+    padding: 4px;
+    font-size: 1.6rem;
+    background-color: rgb(255, 255, 255);
+    color: rgb(95, 95, 95);
+}
```
Comment (manojsarna) [inline my-cart/mycart.css:39]: check for color variables

File: my-cart/mycart.css [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+.navbar-head-logo {
+    color: #ece9f2;
+    display: flex;
+    align-items: center;
+}
```
Comment (manojsarna) [inline my-cart/mycart.css:45]: check

File: my-cart/mycart.css [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+.navbar-login-btn {
+    background-color: rgb(255, 196, 1);
+    padding: .5rem;
+    color: #312546;
+    font-weight: 600;
```
Comment (manojsarna) [inline my-cart/mycart.css:51]: check this

File: my-cart/mycart.css [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+    font-size: 1.1rem;
+    margin: .3rem;
+    width: 8%;
+    height: 2rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: .6rem;
+}
```
Comment (manojsarna) [inline my-cart/mycart.css:60]: if value is lesa than 10px , you can use px values

File: my-cart/mycart.css [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+    align-items: center;
+    display: flex;
+}
+
+.input-text-navbar {
+    border: 1px solid rgb(255, 255, 255);
+    padding: .5rem;
+    width: 25rem;
+    border-radius: 5px 0px 0px 5px;
+}
```
Comment (manojsarna) [inline my-cart/mycart.css:72]: check this

File: my-cart/mycart.css [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+
+.fa-cart-arrow-down {
+    margin: 1rem;
+    font-size: 1.3rem;
+    color: #ece9f2;
+}
```
Comment (manojsarna) [inline my-cart/mycart.css:78]: check this

File: my-cart/mycart.css [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+    display: flex;
+    justify-content: center;
+}
+
+.image-card {
+    width: 80%;
+    height: auto;
+    border-radius: .2rem;
+    margin-left: .3rem;
+}
```
Comment (manojsarna) [inline my-cart/mycart.css:119]: check for px values

File: my-cart/mycart.css [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+
+.card-heading-md-2 {
+    margin-top: 1rem;
+    font-weight: bolder;
+    background-color: #ff9900;
+    padding: .3rem;
+    display: flex;
+    justify-content: center;
+
```
Comment (manojsarna) [inline my-cart/mycart.css:128]: check this

File: my-cart/mycart.css [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+    border-radius: .3rem;
+    align-items: center;
+    display: flex;
+    justify-content: center;
+
+
+}
+.second-card{
+     background-color: #ffffff;
+     display: flex;
```
Comment (manojsarna) [inline my-cart/mycart.css:158]: check color

**Review by manojsarna — APPROVED** [review]
overall its good

do check the comments made!!

other than that good to merge!!

**Pull request merged**